src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml

```diff
@@ -46,7 +46,7 @@
 
             var editor = monaco.editor.create(document.getElementById('@Html.IdFor(x => x.Html)_editor'), settings);
             var textArea = document.getElementById('@Html.IdFor(x => x.Html)');
-            
+
             editor.getModel().onDidChangeContent((event) => {
```

Comment:
How come I see the exact same code that you're adding up on L50?

i.e. don't we already do this?
```
           //for preview
            if(document.getElementById("previewButton")){
                editor.getModel().onDidChangeContent((event) => {
                    textArea.value = editor.getValue();
                    $(document).trigger('contentpreview:render');
                });
            }
```

Reply:
Yes , you are right .. 
I didn't notice the codes 。